platform-core-module/src/main/java/com/softicar/platform/core/module/environment/LiveSystemRevision.java

```diff
@@ -37,23 +38,36 @@ public Optional<String> getName() {
 		return Optional.ofNullable(name);
 	}
 
+	/**
+	 * Returns an Optional with the version of the current revision, or an empty
+	 * Optional if the revision version could not be determined.
+	 *
+	 * @return an Optional with the version of the current revision, or an empty
+	 *         Optional
+	 */
+	public Optional<String> getVersion() {
+
+		return Optional.ofNullable(version);
+	}
+
 	private LiveSystemRevision() {
 
 		try (InputStream inputStream = LiveSystemRevision.class.getResourceAsStream(BUILD_PROPERTIES_FILEPATH)) {
 			if (inputStream != null) {
-				this.name = parseRevision(inputStream);
+				this.name = parseRevision(inputStream, "revision.id");
+				this.version = parseRevision(inputStream, "revision.version");
```

Comment:
parsing the whole properties file multiple times is wastefull
```suggestion
				parseRevisionProperties(inputStream);
```